### DISCUSSION TITLE
GitHub Copilot Chat in VS Code

### DISCUSSION THREAD

**Discussion by MohammedAHamdi** [community/community#86768] [Packages]
### Select Topic Area

Question

### Body

I am using the student package with VS code. After I login into my GitHub account in the IDE, the Copilot icon becomes active and its status is ready. However, it does not seem to be working (no code suggestion), and the Copilot Chat is not working with the package (it asks me to use the 30-day Free Trial).

Any help is appreciated!
<img width="1018" alt="Screen Shot 2024-01-13 at 12 22 22 AM" src="https://github.com/community/community/assets/96801520/805310ac-aea1-47fe-b0db-bcc9590f0d5f">

**Comment by kamranzafar4343** [comment]
Ensure that you have the Copilot extension properly installed and activated within VS Code, and try restarting the IDE to resolve issues with code suggestions and the Copilot Chat feature in the student package.

**Reply by MohammedAHamdi** [reply]
I uninstalled the extension and restarted the IDE. Which one should I select (Install Release Version OR Install Pre-Release Version) to install it again?

<img width="1021" alt="Screen Shot 2024-01-13 at 11 48 24 AM" src="https://github.com/community/community/assets/96801520/02394d37-031c-4c76-b1ac-36fe6cee505b">

**Comment by kamranzafar4343** [comment]
Install release version looks right.

**Reply by MohammedAHamdi** [reply]
I restarted the IDE after installing the extension. The first message shows in the second picture about failing to access the Copilot, I logged out then I logged in into my GitHub account and the Copilot is active now. For the Copilot Chat, it is still the same problem.


<img width="1023" alt="1" src="https://github.com/community/community/assets/96801520/3c5d0f4b-c7e1-458b-ba15-bf0ac9523f7c">
<img width="1022" alt="2" src="https://github.com/community/community/assets/96801520/16f173cc-1f3d-40b1-be84-9f9964dbb0b1">
<img width="1019" alt="3" src="https://github.com/community/community/assets/96801520/e9da6257-ce62-42cb-8c86-620eadc4cf18">

**Comment by kamranzafar4343** [comment]
take help from youtube

**Comment by MohammedAHamdi** [comment]
After checking my GitHub account I noticed that the student package was expired last month (December). I renewed it and it works properly now in VS Code and IntelliJ. Thank you for your help and sorry for my question.
